test/pipeline_test.py

```diff
@@ -44,26 +44,74 @@ def test_drop():
 
     # Drop by columns
     df1 = DplyFrame(pandas_df)
-    output1 = df1 + drop(["col3", "col4"], axis=1)
-    expected1 = pandas_df.drop(["col3", "col4"], axis=1)
+    output1 = df1 + drop("col1", axis=1)
+    expected1 = pandas_df.drop("col1", axis=1)
     pd.testing.assert_frame_equal(output1.pandas_df, expected1)
 
     df2 = DplyFrame(pandas_df)
-    output2 = df2 + drop(columns=["col3", "col4"])
-    expected2 = pandas_df.drop(columns=["col3", "col4"])
+    output2 = df2 + drop(["col3", "col4"], axis=1)
+    expected2 = pandas_df.drop(["col3", "col4"], axis=1)
     pd.testing.assert_frame_equal(output2.pandas_df, expected2)
 
-    # Drop by rows
+    try:
+        DplyFrame(pandas_df) + drop(["col1", "col5"], axis=1)
+    except KeyError:
+        pass
+    else:
+        raise AssertionError("KeyError was not raised")
+
     df3 = DplyFrame(pandas_df)
-    output3 = df3 + drop([0, 1])
-    expected3 = pandas_df.drop([0, 1])
+    output3 = df3 + drop(columns="col1")
+    expected3 = pandas_df.drop(columns="col1")
     pd.testing.assert_frame_equal(output3.pandas_df, expected3)
 
     df4 = DplyFrame(pandas_df)
-    output4 = df4 + drop(index=[2, 3]) + drop(index=[1])
-    expected4 = pandas_df.drop(index=[1, 2, 3])
+    output4 = df4 + drop(columns=["col3", "col4"])
```

Comment:
Same here - trying to test all of the arguments.

Reply:
Your outpu4 is my output8

Reply:
Oh sorry I missed that. Thanks for reminding me!